tests/conftest.py

```diff
@@ -589,12 +589,12 @@ def pytest_collection_modifyitems(session: pytest.Session, config: Config, items
 @fixture
 def python_version(in_container: bool, application_docker_container: Container) -> Optional[str]:
     if in_container:
-        exit_code, output = application_docker_container.exec_run(cmd="python --version")
+        exit_code, output = application_docker_container.exec_run(cmd="python3 --version")
```

Comment:
I'm not sure this makes sense - we test Python 2 containers as well. The idea is that `python` links to the default python version of this container.
How do python2 tests pass like that? o.O

Reply:
True, I changed both mindlessly, intended to change only the second one, as I think ubuntu 22 does not have this alias.

Reply:
kk